build.psm1

```diff
@@ -610,6 +610,22 @@ Fix steps:
         }
     }
 
+    if ($Environment.IsDebian9)
+    {
```

Comment:
Please follow the pattern of `{` being on the same line as `if`

Reply:
fixed